src/queue.jl

```diff
@@ -4,6 +4,11 @@ type Queue{T}
     store::Deque{T}
 end
 
+"""
+    Queue(ty{T}[, blksize::Integer])
+
+This is a constructor to create an object of a `Queue`.The block size `blksize` by default is `1024`.
```

Comment:
Instead of "The block size `blksize` by default is `1024`.", just write `blksize::Integer=1024` in the signature.